source/guides/2021-03-24-AWS-IPI-default-deployment.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: AWS IPI Default Deployment
+authors:
+  - "@elmiko"
+last-updated: "2021-03-24"
+okd-version: "4.7"
+---
+
+This describes the resources used by OpenShift after perfoming an installation
```

Comment:
Shouldn't we refer to OKD here and in the other similar docs?
```suggestion
This describes the resources used by OKD after perfoming an installation
```

Reply:
good spot, i will update